build_runner/bin/build_runner.dart

```diff
@@ -14,26 +15,43 @@ import 'package:build_runner/src/entrypoint/options.dart';
 import 'package:build_runner/src/logging/std_io_logging.dart';
 
 Future<Null> main(List<String> args) async {
-  var logListener = Logger.root.onRecord.listen(stdIOLogListener);
-
   // Use the actual command runner to parse the args and immediately print the
   // usage information if there is no command provided or the help command was
   // explicitly invoked.
-  var commandRunner = new BuildCommandRunner([]);
+  var commandRunner = new BuildCommandRunner([])
+    ..addCommand(new _GenerateBuildScript());
```

Comment:
Any reason we shouldn't add this to the command runner generally?

Reply:
The other place we use this CommandRunner doesn't have the capability of performing this command.